src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -1,6 +1,9 @@
 import { Component, Host, Prop, h } from "@stencil/core";
 import { CSS } from "./resources";
 
+/**
+ * @slot default - A slot for adding pick-list-item elements.
```

Comment:
`A slot for adding calcite-pick-list-item elements.` Or do we want to say `HTMLCalcitePickListItem`?

Reply:
I like the tagname since that's what our users will be typing out.